charts/istio-authorizer/values.yaml

```diff
@@ -62,35 +65,55 @@ serviceAccount:
 ##
 issuerURL: "https://acp.acp-system:8443/default/system"
 
+## Pod resources definition
+resources: {}
+  # requests:
+  #   cpu: 10m
+  #   memory: 48Mi
+  # limits:
+  #   cpu: 100m
+  #   memory: 96Mi
+
+## Define EnvoyFilter
+## Detailed configuration is available here
+## https://istio.io/latest/docs/reference/config/networking/envoy-filter/
+envoyFilter:
+  ## Enables EnvoyFilter
+  ##
+  enabled: false
+  ## Maximum number of requested bytes 
+  ##
+  maxRequestBytes: 8192
+
+## Enables the Envoyfilter for ACP istio-authorizer
+## Configuration options are available here
+## https://istio.io/latest/docs/reference/config/networking/envoy-filter/
+
 ## Credentials for your ACP istio-authorizer
 ##
-##
 clientCredentials:
-  # Specifies whether a kubernetes secret should be created from the provided clientID and clientSecret.
-  # For production setup please create secret manually and use name
-  # parameter to integrate with istio-authorizer
+  ## Specifies whether a kubernetes secret should be created from the provided clientID and clientSecret.
+  ## For production setup please create secret manually and use name
```

Comment:
`please create secret manually` -> `please create the secret (ideally encrypted) manually`

Reply:
fixed